theme/theme.inc

```diff
@@ -75,11 +83,6 @@ function islandora_pdf_preprocess_islandora_pdf(&$variables) {
       $variables['islandora_preview_img'] = theme('image', $params);
       $variables['islandora_content'] = l($variables['islandora_preview_img'], $variables['islandora_full_url'], array('html' => TRUE));
```

Comment:
In the event the `datastream_access` check fails the `islandora_full_url` won't be set which will cause a warning to pop. Could likely render `islandora_preview_img` alone without a link in that case so something like:
```
$variables['islandora_content'] = isset($variables['islandora_full_url']) ? l($variables['islandora_preview_img'], $variables['islandora_full_url'], array('html' => TRUE)) : $variables['islandora_preview_img'];
```